### PR TITLE
Hi, my name is Yura Dmytriv - this is my homework #2.

### PR DESCRIPTION
**Pull request by Yurnero11**

### CODE REVIEW

File: src/main/java/com/lemon/school/Main.java [hunk 1]
```diff
@@ -1,8 +1,70 @@
...
+  }
+
+  //divide - Ділення. Приймає на вхід 2 цілочисельних значення, повертає ціле число
+  public static int divide(int number){
+    return number / 2;
+  }
+
+  //isEven - Чи парне число. Приймає на вхід ціле число, повертає булеве значення
+  //переробить метод або доробить (true or false)
+  public static String isEven(int number){
```
Comment (ivlytvynets) [inline src/main/java/com/lemon/school/Main.java]: should return `boolean`

File: src/main/java/com/lemon/school/Main.java [hunk 2]
```diff
@@ -1,8 +1,70 @@
...
+      // Я не знав чи потрібно робити перевірку на нуль чи ні, тому добавив
+      return "Zero";
+    }else if (number % 2 == 0){
+      return "Even";
+    }else {
+      return "Odd";
+    }
+  }
+
+  public static boolean isEvenMethod2(int number){
```
Comment (ivlytvynets) [inline src/main/java/com/lemon/school/Main.java:64]: correct implementation but you duplicates yourself

File: src/main/java/com/lemon/school/Main.java [hunk 3]
```diff
@@ -1,8 +1,70 @@
...
+
+  }
+
+  //greet - Привітання. Приймає строку, повертає строку. Приклад: Hello Pen
+  public static String greet(String str){
+    return str;
+  }
+
+  //divide - Ділення. Приймає на вхід 2 цілочисельних значення, повертає ціле число
+  public static int divide(int number){
```
Comment (ivlytvynets) [inline src/main/java/com/lemon/school/Main.java]: 2 numbers for input: a, b, res: a / b

**Review by ivlytvynets — CHANGES_REQUESTED** [review]

File: src/main/java/com/lemon/school/Main.java (hunk 1 above)
Comment (Yurnero11) [reply src/main/java/com/lemon/school/Main.java]: corrected

File: src/main/java/com/lemon/school/Main.java (hunk 2 above)
Comment (Yurnero11) [reply src/main/java/com/lemon/school/Main.java:64]: corrected

File: src/main/java/com/lemon/school/Main.java (hunk 3 above)
Comment (Yurnero11) [reply src/main/java/com/lemon/school/Main.java]: corrected

**Review by ivlytvynets — APPROVED** [review]

**Pull request closed without merging**